content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -114,11 +114,25 @@ Once you have a Linux-based Kubernetes control-plane node you are ready to choos
     curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/kube-proxy.yml | sed 's/VERSION/{{< param "fullversion" >}}/g' | kubectl apply -f -
     kubectl apply -f https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-overlay.yml
     ```
-
     {{< note >}}
     If you're using host-gateway use https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-host-gw.yml instead
     {{< /note >}}
 
+    {{< note >}}
+    If you're using a different interface rather than Ethernet (i.e. "Ethernet0 2") on the Windows nodes, you have to modify the line:
+    
+    ```powershell
+    wins cli process run --path /k/flannel/setup.exe --args "--mode=overlay --interface=Ethernet"
+    ```
+    in the flannel-host-gw.yml or flannel-overlay.yml file and specify your interface accordingly.
```

Comment:
nit: this would work better as
```suggestion
    in the `flannel-host-gw.yml` or `flannel-overlay.yml` file and specify your interface accordingly.
```